src/content/en/updates/2020/10/http-cache-partitioning.md

```diff
@@ -0,0 +1,261 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Chrome's http cache partitioning helps with better security and privacy.
+
+{# wf_published_on: 2020-10-01 #}
+{# wf_updated_on: 2020-10-01 #}
+{# wf_featured_image: /web/updates/images/generic/sd-card.png #}
+{# wf_tags: storage-isolation, cache #}
+{# wf_featured_snippet: Chrome's http cache partitioning helps with better security and privacy. #}
+{# wf_blink_components: Blink>Network #}
+
+# Gaining security and privacy by partitioning the cache {: .page-title }
+
+{% include "web/_shared/contributors/agektmr.html" %}
+
+In general, caching can improve performance by storing data so future requests
+for the same data are served faster. For example, a cached resource from the
+network can avoid a round trip to the server. A cached computational result can
+omit the time to do the same calculation.
+
+In Chrome, the cache mechanism is used in various ways and HTTP Cache is one of
+such examples.
+
+## How Chrome's HTTP Cache currently works
+
+As of version 85, Chrome caches resources fetched from the network, using their
+respective resource URLs as the cache key. (A cache key is used to identify a
+cached resource.)
+
+The following example illustrates how a single image is cached and treated in
+three different contexts:
+
+<figure class="attempt-left">
+  <img src="/web/updates/images/2020/10/http-cache-partitioning-1.png">
+  <figcaption>
+    <b>Cache Key</b>: { <code>https://x.example/doge.png</code> }
```

Comment:
What are the braces ({}) for?

Reply:
This expresses a tuple of cache keys.

Reply:
Can you please put the braces inside the `<code>` elements.

Reply:
No. I think it’s consistent in style and it looks ok. Also, if I do that for triple keys, the line overflows or breaks.

Reply:
Most of them do that already. If you don't put them inside the `<code>` element it's not obvious that you're showing a tuple.